internal/pkg/describe/env_test.go

```diff
@@ -38,43 +43,47 @@ func TestEnvDescriber_FilterAppsForEnv(t *testing.T) {
 	}
 	testApp1 := &archer.Application{
 		Project: "testProject",
-		Name: "testApp1",
-		Type: "load-balanced",
+		Name:    "testApp1",
+		Type:    "load-balanced",
 	}
-	testApp3 :=	&archer.Application{
+	testApp3 := &archer.Application{
 		Project: "testProject",
 		Name:    "testApp3",
-		Type: "load-balanced",
+		Type:    "load-balanced",
 	}
 	env1apps := []*archer.Application{testApp1, testApp3}
 	badTags := map[string]string{stack.EnvTagKey: "badEnvName"}
+	goodTags := map[string]string{stack.EnvTagKey: "testEnv1"}
+	mockError := errors.New("some error")
 	testCases := map[string]struct {
-		env *archer.Environment
-		proj *archer.Project
-		apps []*archer.Application
+		//env  *archer.Environment
+		//proj *archer.Project
+		//apps []*archer.Application
 
-		setupMocks func(mocks envDescriberMocks)
-		mockrgClient	func(m *mocks.MockresourceGroupsClient)
+		setupMocks   func(mocks envDescriberMocks)
+		mockrgClient func(m *mocks.MockresourceGroupsClient)
```

Comment:
Looks like you have a `mockResourceGroupsClient` already defined in your `envDescriberMocks` struct [here](https://github.com/aws/amazon-ecs-cli-v2/pull/900/commits/b4de504ee910c104f1174e1f9ad2ded0862b5c4b#diff-6a7ac9331b26ee29d1305016737c4442L17), so we probably don't need `mockrgClient` here as a separate field in the test case, no?